beginner_source/text_sentiment_ngrams_tutorial.py

```diff
@@ -15,26 +14,28 @@
    - YahooAnswers,
    - AmazonReviewPolarity,
    - AmazonReviewFull
+이 포함되어 있습니다.
 
-This example shows how to train a supervised learning algorithm for
-classification using one of these ``TextClassification`` datasets.
+이 예제에서는 ``TextClassification`` 의 데이터셋들 중 하나를 이용해 분류를 위한
+ 지도 학습 알고리즘을 훈련하는 방법을 보여줍니다.
 
-Load data with ngrams
+데이터를 ngrams과 함께 불러옵니다.
 ---------------------
 
-A bag of ngrams feature is applied to capture some partial information
-about the local word order. In practice, bi-gram or tri-gram are applied
-to provide more benefits as word groups than only one word. An example:
+
+bag of ngrams 피쳐는 지역(local) 단어 순서에 대한 부분적인 정보를 포착하기위해 적용합니다.
+실제는 bi-gram이나 tri-gram은 단 하나의 단어를 이용하는 것보다 더 많은 이익을 주기 때문에 적용됩니다.
+예를 들어,
+
 
 ::
 
    "load data with ngrams"
-   Bi-grams results: "load data", "data with", "with ngrams"
-   Tri-grams results: "load data with", "data with ngrams"
+   Bi-grams 결과: "load data", "data with", "with ngrams"
+   Tri-grams 결과: "load data with", "data with ngrams"
 
-``TextClassification`` Dataset supports the ngrams method. By setting
-ngrams to 2, the example text in the dataset will be a list of single
-words plus bi-grams string.
+``TextClassification`` 데이터셋은 ngrams method을 지원합니다. ngrams을 2로 설정하면,
+데이터셋안의 예제 text는 각각의(single) 단어들에 bi-grams 문자열이 더해진 리스트가 될 것입니다.
```

Comment:
```suggestion
데이터셋 안의 예제 텍스트는 각각의(single) 단어들에 bi-grams 문자열이 더해진 리스트가 될 것입니다.
```
글의 다른 부분에서 "text->텍스트"로 번역하여 수정했습니다.